### ISSUE TITLE
is this project still active?

### ISSUE THREAD

**Issue by iambumblehead** [kelektiv/node-cron#578]
There are many open issues and several outstanding PRs. The last change was committed May 26, 2000, ~15 months ago. Is this project still active?

**Comment by suntong** [comment]
Yep, that's the exact reason I start to warn people

> So don't waste your effort -- I found this out only after I found that I've send the same error twice, both followed up on somebody else's previous error report.

in my new issue #579

**Comment by ncb000gt** [comment]
Hey, thanks for the question.

Honestly, for a variety of reasons, I haven't had time to do much in the way of fixes and checking PRs regardless of how much I want to and it's been that way for far too long.

What this project really needs is more maintainers. If anyone is interested in helping maintain the project I'm open to chatting more.

I'm also open to a number of changes to the project that I was more resistant to early on in the project if that # helps. :)

**Comment by suntong** [comment]
I guess, as the starting point, there are many pending issues that need to be fixed.

**Comment by mahyarmirrashed** [comment]
> I guess, as the starting point, there are many pending issues that need to be fixed. 

But like, are there any formal responsibilities? There may be times where work or school gets too much.

**Comment by suntong** [comment]
O~~h, if that's the only concern that you have, then don't worry at all. 

People having a normal life outside maintaining the open source project is _just the norm_, like this project for e.g.,  #579, the first issue was opened on Feb 11, 2020 -- you have 2.5+ years to fix an issue, :) :)

**Comment by mahyarmirrashed** [comment]
> O~~h, if that's the only concern that you have, then don't worry at all.
> 
> People having a normal life outside maintaining the open source project is _just the norm_, like this project for e.g., #579, the first issue was opened on Feb 11, 2020 -- you have 2.5+ years to fix an issue, :) :)

Damn lol, that is a long time. if there are other's on board that would be open to helping me as well, I would be open to it!

**Comment by Hexagon** [comment]
Not saying i'm not biased :)

But give [github.com/hexagon/croner](github.com/hexagon/croner) a try.

It's active (I'm looking for bugs to fix), compatible all the way from Node 8 to ES-browser-module implementation, has TS definitions etc.

Fiddle [https://jsfiddle.net/7n15m8f4/](https://jsfiddle.net/7n15m8f4/)

**Comment by soltoshi-coin** [comment]
You should add a hint to the main page that this project is currently unmainted. Saves a lot of time for most people, this is as unusable as it gets atm. with all the bugs.

**Comment by ncb000gt** [comment]
I'm happy to add other maintainers to the project. Clearly I haven't had enough time to maintain this project with everything else going on.

There are a few options as I see them:
* Other people step in and become direct contributors helping to review and land PRs and shape the direction of the project **best case**
* Pick from the plethora of other cron libraries for node out there.

I have always held out hope that I'd be able to get back to this project, but as it is my backend tooling has moved away from node so have far less time to dedicate to it as a result.

**Comment by mryellow** [comment]
No. It has critical faults and no one is publishing fixes.

**Comment by ncb000gt** [comment]
@mryellow I shared that I am looking for help with the project. I'd be happy to look at a pr.

Thanks!

**Comment by mryellow** [comment]
Problem is it's just such a big refactor for some of those functions, they essentially need deleting and starting again.

So without an active maintainer, it's easier to just move on.

**Comment by iambumblehead** [comment]
In practice, there are still several un-acknowledged PRs

**Comment by intcreator** [comment]
I'm taking a more active role in maintaining this project. a lot of issues are duplicates so my first objective is closing those to narrow down what issues actually need attention. I'll take a look at PRs next

going to close this since the answer to the original question is yes
